2024Robot_Team3484/src/main/include/Constants.h

```diff
@@ -1,7 +1,22 @@
 #ifndef CONSTANTS_H
 #define CONSTANTS_H
+#include <units/angular_velocity.h>
+namespace ShooterConstants {
+    constexpr int MOTOR_LEFT_PORT = 0;
+    constexpr int MOTOR_RIGHT_PORT = 1;
 
-namespace ShooterConstants {}
+    constexpr double P = 0;
+    constexpr double I = 0;
+    constexpr double D = 0;
+    constexpr double FF = 0;
+    constexpr double RPM_Window = 50;
+
+    //constexpr bool IsLoaded = true;
+    constexpr bool MOTER_LEFT_INVERTED = true;
+
+
+    constexpr units::revolutions_per_minute_t RPM = 2000_rpm;
```

Comment:
What is the purpose of `RPM`? I suggest renaming this variable to be more descriptive of what purpose this variable provides.